controllers/controllers.js

```diff
@@ -0,0 +1,35 @@
+const {getAllCustomers, getOneCustomer, addNewCustomers} = require('./models/models')
+
+const getAllCustomersController = async (request, response) => {
+   try{const allCustomers = await getAllCustomers()
+
+    return response.send(allCustomers)
+   } catch(error){
+        return response.sendStatus(500)
+   }
+}
+
+const getOneCustomerController = async (request, response) => {
```

Comment:
nitpick: remember to rename these functions to be more explicit with what the function does. For example this one could be `getCustomerById`